README.md

```diff
@@ -112,12 +112,12 @@ t.Log(people)
 Installation
 
 ```bash
-go get -u "github.com/scylladb/gocqlx/v3/cmd/schemagen"
+go install github.com/scylladb/gocqlx/v2/cmd/schemagen@latest
```

Comment:
we should use latest version which is v3

Reply:
```suggestion
go install github.com/scylladb/gocqlx/v3/cmd/schemagen@latest
```